include/verilated.mk.in

```diff
@@ -283,9 +283,9 @@ $(VK_GLOBAL_OBJS): %.o: %.cpp
 # Precompile a header file
 # PCH's compiler flags must match exactly the rules' above FAST/SLOW
 # arguments used for the .cpp files, or the PCH file won't be used.
-%.fast.gch: %
+%.fast$(CFG_GCH_IF_CLANG): %
```

Comment:
The output filename should always end in .gch, so the previous code was correct.

Reply:
Are you sure? The comment for CFG_GCH_IF_CLANG suggests otherwise, and it doesn't work if I remove `$(CFG_CGH_IF_CLANG)` (`No rule to make target 'Vtop__pch.h.fast'`).

Reply:
I'm sure the original intent of CFG_CGH_IF_CLANG was only to change what was needed on command line and the compiler-generated file itself always had .gch.  As to what's needed in your case, I don't know.